modules/core/src/v2/wallet.ts

```diff
@@ -484,6 +487,53 @@ export interface DownloadKeycardOptions {
   backupKeyID?: string;
 }
 
+// #region TSS interfaces
+
+export interface PrebuildTransactionWithIntentOptions {
+  reqId?: RequestTracer;
+  intentType?: string;
+  sequenceId?: string;
+  recipients?: {
+    address: string;
+    amount: string | number;
```

Comment:
I think here we can unify in just one type. This is just a comment and may need more analysis.

Reply:
i think string is better than number here.

clients have been experience a couple of issues with numbers being too large (i think AVAX), which produced some unexpected results due to rounding.

safer to have this be a string and use `BigNumber` to perform operations on it

Reply:
I will have to convert it to string, since im just passing what i got from the params of https://github.com/BitGo/BitGoJS/blob/master/modules/core/src/v2/wallet.ts#L386

Also i may have to convert it to this 'Amount' type https://github.com/BitGo/bitgo-microservices/blob/210059943d402e8abdc88dca08202768b462f4ae/packages/common-interface/src/interface/api.ts#L305

Reply:
@zahin-mohammad Seems that old `recipients:{address:string, amount:string|number}[]` are not compatible with the txRequest expected types `recipients: { address:Address, amount: Amount}[]`

Reply:
We don't necessarily need to do it here, but as long as we have a way of translating things to the intent when it comes time to interact with the WP api.